inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/reification/WikiDataReification.java

```diff
@@ -391,25 +396,53 @@ public void upsertQualifier(KnowledgeBase kb, KBQualifier aQualifier)
     @Override
     public List<KBQualifier> listQualifiers(KnowledgeBase kb, KBStatement aStatement)
     {
-        String statementId = aStatement.getStatementId();
-        if (statementId == null) {
-            log.error("No statementId");
-            return null;
-        }
-        else {
-            List<Statement> qualifierStatements = getQualifiersById(kb, statementId);
-            List<KBQualifier> qualifiers = new ArrayList<>();
-            for (Statement qualifierStatement : qualifierStatements) {
-                KBHandle property = new KBHandle();
-                property.setIdentifier(qualifierStatement.getPredicate().stringValue());
-                Value value = qualifierStatement.getObject();
-                KBQualifier qualifier = new KBQualifier(aStatement, property, value);
+        List<KBQualifier> qualifiers = new ArrayList<>();
+        String QUERY = String.join("\n",
+            "SELECT DISTINCT ?p ?o ?l WHERE {",
+            "  ?id ?p ?o .",
+            "  ?p  ?pLABEL ?l.",
```

Comment:
For consistency with the rest of the KB service, better use the following to get the label which prefers English labels and also retrieves data for which there is no label (happens):

```
OPTIONAL { 
    ?s ?pLABEL ?l .
    FILTER(LANG(?l) = \"\" || LANGMATCHES(LANG(?l), \"en\"))
} 
```